dependencies.props

```diff
@@ -47,17 +50,18 @@
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>PlatformPackageVersion</ElementName>
-      <PackageId>Microsoft.NETCore.Platforms</PackageId>
+      <ElementName>CoreFxExpectedPrerelease</ElementName>
```

Comment:
Can you explain to me what this part of the diff does?  Is it no longer looked at Latest_Packages.txt and instead looking at something else (I ask because I depend on this working and if this triggers it looking at another file I need to augment my local tasks in the build from source meta repository to handle that.